packages/victory-core/src/victory-primitives/background.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+import CommonProps from "../victory-util/common-props";
+import Rect from "./rect";
+import Circle from "./circle";
+
+const getRangeBounds = (props) => ({ x: props.range.x[0], y: props.range.y[1] });
+const getHeight = (props) => (props.polar ? props.range.y[1] : props.range.y[0] - props.range.y[1]);
+const getWidth = (props) => props.range.x[1] - props.range.x[0];
+
+const Background = (props) => {
+  return props.polar
+    ? React.cloneElement(props.circleComponent, {
+        ...props.events,
+        style: props.style && props.style.background,
+        role: props.role,
+        shapeRendering: props.shapeRendering,
+        cx: "50%",
+        cy: "50%",
+        r: getHeight(props)
+      })
+    : React.cloneElement(props.rectComponent, {
+        ...props.events,
+        style: props.style && props.style.background,
+        role: props.role,
+        shapeRendering: props.shapeRendering,
+        x: getRangeBounds(props).x,
+        y: getRangeBounds(props).y,
+        width: getWidth(props),
+        height: getHeight(props)
+      });
+};
+
+Background.propTypes = {
+  ...CommonProps.baseProps,
+  categories: PropTypes.oneOfType([
```

Comment:
I think it might make more sense for this component to extend from `CommonProps.primitiveProps` as other primitive components do. `range` would then be taken from the `scale` prop that `VictoryChart` provides like `scale.x.range()`.

Reply:
Ah! Since it's getting the full set of props passed from victory-chart, should we do both baseProps and primitiveProps?

Reply:
I think it should only be getting a subset of the props. Right now you're treating it like a normal child component of VictoryChart, but I think it's too much of a special case. It should have a special set of props calculated